packages/api/src/utils/schemas.ts

```diff
@@ -27,16 +27,17 @@ const zodRollupEnums = [
   "zksync",
   "mode",
   "zora",
+  "unknown",
```

Comment:
"unknown" would make sense for rollups.

For categories I believe "other" or "uncategorized" are better choices.